### PR TITLE
METRON-208 Add Google Analytics tags to Metron Home Pages

### PR DESCRIPTION
**Pull request by iraghumitra**
-Added Google Analytics support to Metron website
-Added screen text for few sections in the docs and community pages

### CODE REVIEW

File: site/diff [hunk 1]
```diff
@@ -0,0 +1,186 @@
+diff --git a/site/_includes/footer.html b/site/_includes/footer.html
```
Comment (cestella) [inline site/diff]: Did this file get committed accidentally?  Probably don't want the diff here.

**Comment by mmiklavc** [comment]
(Also commented on METRON-173, but including here since this PR is currently open)

Ran the local jekyll server for the site - everything looks great. I am still seeing a number of missing file references show up in the logs while browsing, e.g.:
```
ERROR `/images/footer-overlay-bg.jpg' not found.
ERROR `/images/close-icon.png' not found.
ERROR `/wp-content/themes/hortonworks/images/overlay-close.png' not found.
ERROR `/wp-content/themes/hortonworks/images/logo1.png' not found.
ERROR `/images/loader1.gif' not found.
ERROR `/images/footer-overlay-bg.jpg' not found.
ERROR `/images/close-icon.png' not found.
ERROR `/wp-content/themes/hortonworks/images/overlay-close.png' not found.
ERROR `/wp-content/themes/hortonworks/images/logo1.png' not found.
ERROR `/images/loader1.gif' not found.
```

**Comment by iraghumitra** [comment]
-Removed the diff file that accidentally got in
-Removed all the unused imports as they were throwing bunch of errors
-Added Ryan as PMC in contributors page

**Comment by cestella** [comment]
+1 looks good to me

**Comment by merrimanr** [comment]
My apache id is incorrect.  After @iraghumitra makes the change +1

**Comment by iraghumitra** [comment]
Corrected apache id of Ryan

**Comment by iraghumitra** [comment]
Corrected Ryan's apache id

**Pull request closed without merging**